packages/theme-patternfly-org/scripts/getHtmlWebpackPlugins.js

```diff
@@ -1,23 +1,26 @@
 const path = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const { googleAnalyticsID, algolia } = require(`${process.cwd()}/patternfly-docs.config`);
+const { prerender } = require('./prerender');
 
-module.exports = (routes, isProd) => 
-  Object.entries(routes).map(([url, { title }]) => 
-    new HtmlWebpackPlugin({
-      template: path.resolve(__dirname, '../templates/html.ejs'),
-      filename: `${url}/index.html`.replace(/^\/+/, ''),
-      title: `PatternFly 4${title ? ` • ${title}` : ''}`,
-      templateParameters: {
-        prerendering: isProd ? require('./prerender')(url) : 'Loading...',
-        // Don't use GA in dev mode
-        googleAnalyticsID: isProd ? googleAnalyticsID : false,
-        algolia
-      },
-      scriptLoading: 'defer',
-      inject: false
-    }),
-  ).concat(
+async function getHtmlWebpackPlugin(url, isProd, title) {
+  return new HtmlWebpackPlugin({
+    template: path.resolve(__dirname, '../templates/html.ejs'),
+    filename: `${url}/index.html`.replace(/^\/+/, ''),
+    title: `PatternFly 4${title ? ` • ${title}` : ''}`,
```

Comment:
In the preview i am not seeing the title correctly for the browser tab

Reply:
Yep, it seems the HtmlWebpackPlugin isn't respecting this option. I'll fix it!